src/components/BulkEnrollmentPage/CourseSearchResults.jsx

```diff
@@ -25,20 +27,49 @@ export const BaseCourseSearchResults = ({
     pageCount: searchResults?.nbPages || 1,
   }), [searchResults?.pageIndex]);
 
+  const [modalOpen, setModalOpen] = useState(false);
+  const [selectedCourseRuns, setSelectedCourseRuns] = useState([]);
+  const [showToast, setShowToast] = useState(false);
```

Comment:
Don't mind my snooping... but you might consider using the `Toasts` feature that exists globally throughout this application, e.g.:

```jsx
import { ToastsContext } from '../Toasts';

const Example = () => {
  const { addToast } = useContext(ToastsContext);

  useEffect(
    () => addToast('The toastiest of all toast messages 🍞'),
    [],
  );
};
```